Palo Alto Networks/paloalto-ngfw/ingest/parser.yml

```diff
@@ -184,7 +184,7 @@ pipeline:
     external:
       name: dsv.parse-dsv
       properties:
-        input_field: original.message
+        input_field: "{{ original.message.replace('\\\\\\\"', '\\\"') }}"
```

Comment:
Why did you add this replacement?

Reply:
There was a problem in the `globalprotect_csv_2.json` event, The value `Microsoft Windows 10 Enterprise, 64-bit` is divided into two, causing a column misalignment. and this's because of `\\\\\\\"`.